.github/workflows/sonarqube.yml

```diff
@@ -16,6 +16,12 @@ on:
 jobs:
   build:
     if: github.repository != 'micronaut-projects/micronaut-project-template'
+    env:
```

Comment:
FYI this workflow is being removed since we are running coverage in the main Gradle workflow. See https://github.com/micronaut-projects/micronaut-project-template/pull/342